apps/pie-docs/README.md

```diff
@@ -31,6 +31,22 @@
 - Our unit tests are written using [Jest](https://jestjs.io/)
 - Our accessibility tests are written using [Axe with Playwright](https://playwright.dev/docs/accessibility-testing)
 
+## Drafts
+
+When building a page that is not yet ready for production we can mark the page as a `draft` by adding `draft: true` to the page front matter. This will allow 11ty to build the page during development mode but will exclude the page from builds during production.
+
+```
+---
+title: Test Page
+description: Test page
+draft: true
+---
+```
+
+To create a new section that will be in draft mode, each page in the section will need to have `draft: true` added to the page front matter.
```

Comment:
In my experience this isn't ideal. Especially if you want to use the latest-change date to be reflected in the post/page.

Note that Eleventy supports frontmatter from a **Directory Data File** (data applies to all templates in posts/subdir/*).

See: https://www.11ty.dev/docs/data-template-dir/

Just something to consider.

Reply:
👍